decision-trees.Rmd

```diff
@@ -396,13 +394,13 @@ dt_tune_metrics %>%
   facet_wrap(~min_n)
 ```
 
-So generally it's looking like lower values of of cost complexity are leading to better performing models, and the minimum sample size for a terminal node is looking best at 21 or 30. Let's look at our best models in a more tabular form. This time we'll use `show_best` to show our best hyperparameter combinations.
+It looks like lower values of cost complexity are leading to better performing models, and the minimum sample size for a terminal node is looking best at 21 or 30. Let's look at our best models in a more tabular form. This time we'll use `show_best()` to show our best hyperparameter combinations.
 
 ```{r }
 show_best(dt_tune_fit, metric = "roc_auc")
 ```
 
-And we see a little bit different picture here. Our best model has a minimum $n$ of 11 with a relatively higher cost complexity. But the amount this model is "better" is trivial and could easily be due to sampling variability. All of the rest of the best models have the same minimum $n$, with the cost complexity playing essentially no role. This may lead us to consider *not* pruning by the cost complexity parameter at all. Let's use all the results again to look at the minimum $n$ a little closer. We'll filter for a very low cost complexity.
+We see a somewhat different picture here. Our best model has a minimum $n$ of 11 with a relatively higher cost complexity. But the amount this model is "better" is trivial and could easily be due to sampling variability. All of the rest of the best models have the same AUC performance, with the cost complexity playing essentially no role. This may lead us to consider *not* pruning by the cost complexity parameter at all. Let's use all the results again to look at the minimum $n$ a little closer. We'll filter for a very low cost complexity.
```

Comment:
The remaining best models all have *different* min n, so I'm assuming this was a typo, but I'm not actually sure what exactly you wanted to highlight here. I made an educated guess but wanted to double check. 🙃 

![Screen Shot 2020-12-30 at 1 30 53 PM](https://user-images.githubusercontent.com/32279312/103383034-19a1e900-4aa6-11eb-9136-4d586c1c9360.png)